src/controllers/mainController.ts

```diff
@@ -1178,6 +1187,19 @@ export default class MainController implements vscode.Disposable {
 
 			await this.sanitizeConnectionProfiles();
 
+			if (e.affectsConfiguration(Constants.cmdObjectExplorerGroupBySchemaFlagName)) {
+				this._objectExplorerProvider.rootNodeConnections.forEach(async (conn) => {
+					if (this.connectionManager.isActiveConnection(conn)) {
+						const rootNodes = await this._objectExplorerProvider.getChildren();
+						rootNodes.forEach(async (rootNode: TreeNodeInfo) => {
+							if (rootNode.connectionInfo.accountId === conn.accountId) {
```

Comment:
What is this check for? Shouldn't we be able to just refresh the root nodes themselves regardless and have it pick up the new grouping that way?